config.go

```diff
@@ -35,6 +35,7 @@ const (
 	defaultImagePath            = "/usr/share/clear-containers/clear-containers.img"
 	defaultHypervisorPath       = "/usr/bin/qemu-lite-system-x86_64"
 	defaultProxyURL             = "unix:///run/cc-oci-runtime/proxy.sock"
+	defaultPauseBinPath         = "/tmp/bundles/pause_bundle/rootfs/bin/pause"
```

Comment:
@sboeuf sorry if my answer was confusing but I'd like:

1. The pause **bundle** path to be optionally defined from `config.toml` and use the default `/var/lib/clearcontainers/runtime/bundles/pause_bundle` if not specified. In any case it must be an aboslute path, obviously.
2. Use `bin/pause` as the non configurable relative path to the pause binary.

Does that make more sense ?